account_invoice_mass_sending_direct_print/models/res_config_settings.py

```diff
@@ -0,0 +1,12 @@
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    invoice_is_direct_print = fields.Boolean(
+        string="Invoice Is Direct Print",
+        related="company_id.invoice_is_direct_print",
+        readonly=False,
+        help="Allow invoice to be printed directly when 'Send and Print' is clicked",
```

Comment:
confusing

better say things explicitly then if you are not referring to a specific button: the `Send & Print` wizard